### PR TITLE
chore: Remove unused lib-v1 pages & cleanup errors in fragment imports.

### PR DESCRIPTION
**Pull request by jimblanc**
#### Description of changes:
This change removes unused lib-v1 pages to prevent the routes from being generated and fixes several errors in fragment imports that were impacting other platforms.

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://link.com)` 
            HTML: `<a href="https://link.com">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/directory/directory.mjs [hunk 1]
```diff
@@ -1445,7 +1445,7 @@ export const directory = {
           {
             title: 'Get file properties',
             route: '/lib-v1/storage/get-properties',
-            filters: ['flutter', 'js']
+            filters: ['js']
```
Comment (jimblanc) [inline src/directory/directory.mjs:1448]: This page should not exist for flutter lib-v1, current docs: https://docs.amplify.aws/lib-v1/storage/get-properties/q/platform/flutter/

File: src/directory/directory.mjs [hunk 2]
```diff
@@ -1455,10 +1455,10 @@ export const directory = {
           {
             title: 'Copy files',
             route: '/lib-v1/storage/copy',
-            filters: ['js', 'flutter']
```
Comment (jimblanc) [inline src/directory/directory.mjs:1458]: This page should not exist for flutter lib-v1, current docs: https://docs.amplify.aws/lib-v1/storage/copy/q/platform/flutter/

File: src/pages/lib-v1/analytics/create-custom-plugin/q/platform/[platform].mdx
Comment (jimblanc) [inline src/pages/lib-v1/analytics/create-custom-plugin/q/platform/[platform].mdx:1]: Functionality described by this page does not exist in Dev Preview. Unused by other platforms.

File: src/pages/lib-v1/analytics/personalize/q/platform/[platform].mdx
Comment (jimblanc) [inline src/pages/lib-v1/analytics/personalize/q/platform/[platform].mdx:1]: Functionality described by this page does not exist in Dev Preview. Unused by other platforms.

File: src/pages/lib-v1/analytics/storing/q/platform/[platform].mdx
Comment (jimblanc) [inline src/pages/lib-v1/analytics/storing/q/platform/[platform].mdx:1]: Functionality described by this page does not exist in Dev Preview. Unused by other platforms.

File: src/pages/lib-v1/analytics/streaming/q/platform/[platform].mdx
Comment (jimblanc) [inline src/pages/lib-v1/analytics/streaming/q/platform/[platform].mdx:1]: Functionality described by this page does not exist in Dev Preview. Unused by other platforms.

File: src/pages/lib-v1/storage/copy/q/platform/[platform].mdx [hunk 3]
```diff
@@ -6,7 +6,3 @@ export const meta = {
 import js6 from '/src/fragments/lib-v1/storage/js/copy.mdx';
 
 <Fragments fragments={{ js: js6 }} />
-
-import flutter0 from '/src/fragments/lib/storage/flutter/copy.mdx';
-
-<Fragments fragments={{ 'flutter': flutter0 }} />
```
Comment (jimblanc) [inline src/pages/lib-v1/storage/copy/q/platform/[platform].mdx:12]: Added by mistake at some point, Flutter does not expose copy in lib-v1 route.

File: src/pages/lib-v1/storage/get-properties/q/platform/[platform].mdx [hunk 4]
```diff
@@ -6,7 +6,3 @@ export const meta = {
 import js6 from '/src/fragments/lib-v1/storage/js/get-properties.mdx';
 
 <Fragments fragments={{ js: js6 }} />
-
-import flutter0 from '/src/fragments/lib/storage/flutter/get-properties.mdx';
-
-<Fragments fragments={{ flutter: flutter0 }} />
```
Comment (jimblanc) [inline src/pages/lib-v1/storage/get-properties/q/platform/[platform].mdx:12]: Added by mistake at some point, Flutter does not expose get-properties in lib-v1 route.

File: src/pages/lib-v1/storage/getting-started/q/platform/[platform].mdx
Comment (jimblanc) [inline src/pages/lib-v1/storage/getting-started/q/platform/[platform].mdx:1]: This file was mistakenly updated to point to `lib` routes for other platforms. Here's the final diff with main after my change:

```
b0f1d8581327:amplify-docs jablanch$ git diff main chore/fragment-cleanup -- src/pages/lib-v1/storage/getting-started/q/platform/\[platform\].mdx
diff --git a/src/pages/lib-v1/storage/getting-started/q/platform/[platform].mdx b/src/pages/lib-v1/storage/getting-started/q/platform/[platform].mdx
index bc207b52b..7ca6d3940 100644
--- a/src/pages/lib-v1/storage/getting-started/q/platform/[platform].mdx
+++ b/src/pages/lib-v1/storage/getting-started/q/platform/[platform].mdx
@@ -26,3 +26,7 @@ import android2 from '/src/fragments/lib-v1/storage/native_common/getting-starte
 import flutter3 from '/src/fragments/lib-v1/storage/native_common/getting-started/common.mdx';

 <Fragments fragments={{ flutter: flutter3 }} />
+
+import js6 from '/src/fragments/lib-v1/storage/js/getting-started.mdx';
+
+<Fragments fragments={{ js: js6 }} />
```

**Review by AllanZhengYP — APPROVED** [review]
Thank you for cleaning this! @jimblanc

**Pull request merged**